documentation/modules/configuring/con-config-high-volume-messages.adoc

```diff
@@ -0,0 +1,214 @@
+// Module included in the following assemblies:
+//
+// assembly-config-high-volumes.adoc
+
+[id='con-high-volume-config-properties-{context}']
+= Handling high volumes of messages
+
+[role="_abstract"]
+Operations like tracking web activity can generate a high volume of messages. 
+But even a data source with moderate throughput can produce a high volume of messages if it has a lot of existing data. 
+If your Strimzi deployment needs to handle a high volume of messages, you can use configuration options to optimize for throughput and latency. 
+
+Producer and consumer configuration helps to control the size and frequency of requests to Kafka brokers. 
+For more information on the configuration options, see the following:
+
+* xref:con-producer-config-properties-throughput-{context}[] 
+* xref:con-consumer-config-properties-throughput-{context}[]  
+
+You can also use the same configuration options with the Kafka Connect runtime producers and consumers used by Kafka Connect and MirrorMaker 2.0 source connectors.
+Consumers are used by source connectors to retrieve data from an external data system for Kafka Connect or from a source Kafka cluster for MirrorMaker 2.0.
+Producers are used by source connectors to send messages to a target Kafka cluster.
```

Comment:
This is a bit confusing. MM2 _source_ connector consumes from source cluster and produces to target cluster. Connect _source_ connector produces messages to Kafka cluster and _sink_ connector consumes messages from Kafka. I think we need to clearly separate these because of the confusing naming.

Reply:
I edited this based on Mickael's comments to make clear the source/sink and MM2 and KC